tsup.config.ts

```diff
@@ -0,0 +1,21 @@
+import { defineConfig } from 'tsup';
+
+export default defineConfig([
+  {
+    clean: true,
+    entry: ['./src/**/!(*.{d,test}).{js,jsx,ts,tsx}'],
+    format: ['cjs', 'esm'],
+    dts: true,
+    esbuildOptions(options, context) {
+      options.platform = 'node';
+    },
+  },
+  {
+    clean: true,
+    entry: ['bin'],
```

Comment:
We should just move the file into `src` then?

Reply:
Done